enterprise/backend/src/metabase_enterprise/transforms/api.clj

```diff
@@ -238,7 +238,7 @@
   [{:keys [id]} :- [:map
                     [:id ms/PositiveInt]]]
   (log/info "run transform" id)
-  (api/check-superuser)
+  (api/write-check :model/Transform id)
```

Comment:
Feels weird to use `write-check` here, because we're *using* the model, not updating it.